src/Microsoft.AspNet.Mvc.ModelBinding/Binders/MutableObjectModelBinder.cs

```diff
@@ -334,29 +337,6 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
                 if (requiredValidator != null)
```

Comment:
just to be clear: this change leaves `[Required]` also implicitly meaning `BindingBehavior.Required`?